dist/gmusic.js

```diff
@@ -250,7 +250,7 @@ proto.rating = {
     // Selected thumbs up:
     // <paper-icon-button icon="sj:thumb-up-outline" data-rating="5" role="button" tabindex="0" aria-disabled="false" class="x-scope paper-icon-button-0" title="Undo thumb-up" aria-label="Undo thumb-up"></paper-icon-button>
     // jscs:enable maximumLineLength
-    return el.getAttribute('aria-label').indexOf('Undo') !== -1;
+    return el.__data__.icon === 'thumb-up' || el.__data__.icon === 'thumb-down';
```

Comment:
Weird stuff. Alright, looks good to me but we should update the comment above. How about something like:

```js
     // jscs:disable maximumLineLength
     // Unselected thumbs down:
     // <paper-icon-button icon="sj:thumb-up-outline" data-rating="5" role="button" tabindex="0" aria-disabled="false" class="x-scope paper-icon-button-0" title="Thumb-up" aria-label="Thumb-up"></paper-icon-button>
     //   el.__data__.icon = 'sj:thumb-down-outline';
     // Selected thumbs up:
     // <paper-icon-button icon="sj:thumb-up-outline" data-rating="5" role="button" tabindex="0" aria-disabled="false" class="x-scope paper-icon-button-0" title="Undo thumb-up" aria-label="Undo thumb-up"></paper-icon-button>
     //   el.__data__.icon = 'thumb-up';
     // jscs:enable maximumLineLength
     // DEV: We avoid using English-only strings (e.g. "Undo") for i18n support
     return el.__data__.icon === 'thumb-up' || el.__data__.icon === 'thumb-down';
```